From 4d5426a570c2820c5894a61b52e3dc147e4e7925 Mon Sep 17 00:00:00 2001
From: Joel Sing <joel@sing.id.au>
Date: Tue, 26 Sep 2017 04:10:32 +1000
Subject: [PATCH] [release-branch.go1.4] time: Skip TestParseInLocation test.

This test is now failing due to tz data updates (see issue #19457) - skip the test
since it does not impact the use of Go 1.4 for bootstrapping.

Similar to issue #17545.

Updates #20672

Change-Id: I8017159fa2a59b052c49fda7d50be7a863e0a08d
Reviewed-on: https://go-review.googlesource.com/66073
Reviewed-by: Ian Lance Taylor <iant@golang.org>
diff -pruN '--exclude=*.orig' go~/src/time/format_test.go go/src/time/format_test.go
--- go~/src/time/format_test.go	2015-09-23 06:37:37.000000000 +0000
+++ go/src/time/format_test.go	2018-05-29 20:30:17.862652829 +0000
@@ -184,6 +184,8 @@ func TestParse(t *testing.T) {
 }
 
 func TestParseInLocation(t *testing.T) {
+	t.Skip("skipping test for Go 1.4; Issue 19457")
+
 	// Check that Parse (and ParseInLocation) understand that
 	// Feb 01 AST (Arabia Standard Time) and Feb 01 AST (Atlantic Standard Time)
 	// are in different time zones even though both are called AST
